libr/cons/dietline.c

```diff
@@ -1249,7 +1256,7 @@ R_API const char *r_line_readline_cb(RLineReadCallback cb, void *user) {
 				}
 				if (buf[0] == 0x5b) { // [
 					switch (buf[1]) {
-					case 0x33: // supr
+					case '3': // supr
```

Comment:
not the same, buf is unsigned, chars are signed, and knowing that 0x33 is supr is more meaningful than using '3', which is clear that is the same, but its confusing

Reply:
- signednes of chars is arch dependent
- '3' is a character constant, which, according to the C standard, has type 'int'